README.md

```diff
@@ -1,150 +1,197 @@
+# Nix Postgres Prototype
 
-This repository contains **experimental code** to package PostgreSQL using
-**[Nix]**, and tooling and infrastructure for deploying it. Read the rest of
-this README for some basic overview of the high level components.
+This repository contains **experimental code** to package PostgreSQL using nix.
 
-Don't know Nix? Want to understand some of the thinking here? To learn about Nix
-and some of the design constraints this repository are under, please see the
-[`docs/`](./docs/) directory which should help get you up to speed.
+## Install Nix
+Follow installation guide at [installation](./docs/start-here.md).
 
-If you want to install Nix and play along quickly, check out the
-[Start Here](./docs/00-START-HERE.md) page.
+# Evaluation Criteria
 
-> ⚠️ 🚧 **NOTE** 🚧 ⚠️ &mdash; This repository is **EXPERIMENTAL**. It is not an
-> official Supabase project, and may be abandoned or changed at any moment;
-> there is no commitment to any kind of external third party support at this
-> time.
+## Building multiple versions of PostgreSQL: 14, 15 and future versions
 
-[Nix]: https://nixos.org
+Supported major versions of Postgres are defined in [flake.nix basePackages](https://github.com/supabase/nix-postgres/blob/648e31bc7b629c6644b07ad378d04d3334403d78/flake.nix#L245-L246).
 
-## Fundamental components of this repository
-
-This repository contains a few major high level components, which are outlined below.
-
-### PostgreSQL Binary Distributions
-
-The following PostgreSQL releases are packaged up, along with extensions (read
-below) in order to create "fully self contained" binary distributions. In order to get ahold of them, you can use `nix build`:
-
-```
+Each major version can be built independently.
+```bash
 nix build .#psql_14/bin -o result-14
 nix build .#psql_15/bin -o result-15
 ```
 
-This will create two symlinks named `result-14` (resp. `result-15`). You can use the resulting symlinks as a binary distribution; as if they were unpacked tarballs.
+Builds Postgres 14 & 15 with all of our extensions and symlinks the resulting build artifaces e.g. `psql`, `pg_dump` etc at `result-15` and `result-15` respectively.
 
-### PostgreSQL Docker Images
 
-From the binary images generated by `nix build`, we also create Docker images. You can create
-and load an image like so:
+On first run these commands will take a long time to complete. Intermediate results are cached and reused leading to significantly reduced built times on subsequent runs.
 
-```
-nix build .#psql_15/docker -o result-15
-docker load -i ./result-15
-```
+The Postgres build itself is defined upstream in [nixpkgs#postgresqlXX](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/databases/postgresql.nix). That distribution is frequently updated and historically has released minor and major updates within 10 days. We can optionally override the version number or vendor the derivation to support additional versions.
```

Comment:
Correction since the link is to the NixOS module, not the postgresql build.

```suggestion
The Postgres build itself is defined upstream in [nixpkgs#postgresqlXX](https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/sql/postgresql/default.nix). That distribution is frequently updated and historically has released minor and major updates within 10 days. We can optionally override the version number or vendor the derivation to support additional versions.
```